views/index.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
+  </head>
+  <body>
+    <!-- <a href='/start'>New Game</a> -->
```

Comment:
Be careful about leaving commented code in, especially when the next line does the same thing